commons-executors/src/main/java/com/palantir/common/concurrent/NamedThreadFactory.java

```diff
@@ -44,20 +42,10 @@ public NamedThreadFactory(String prefix) {
         this(prefix, true);
     }
 
-    /**
-     * Creates a new thread factory that will construct threads with names like
-     * <i>prefix</i>-<i>seqence_number</i>, where sequence number counts up from zero,
-     * and prefix is as specified in the constructor.
-     *
-     * @param prefix The prefix for each constructed thread.
-     * @param isDaemon {@code true} iff the constructed threads should be daemon threads.
-     */
-    @SuppressWarnings("deprecation") // No reasonable way to pass a TaggedMetricRegistry
     public NamedThreadFactory(String prefix, boolean isDaemon) {
         this.prefix = prefix;
         this.isDaemon = isDaemon;
-        this.threadFactory = MetricRegistries.instrument(
-                SharedTaggedMetricRegistries.getSingleton(), Executors.defaultThreadFactory(), prefix);
```

Comment:
This was also resulting in some double-wrapping, since this is lower-level than `PTExecutors`.

Reply:
Are you sure this is double-wrapped? The threadfactory metrics describe the number of live platform threads in a pool which may be idle, where the executorservice metrics describe active/submitted tasks (without context on the state of the underlying pool).

Reply:
You're right, the wrapping here is around the entire thread task, not the executor tasks.